index.html

```diff
@@ -9,13 +9,64 @@
   </head>
   <body>
     <h1 class="title">Front-End Tools</h1>
-    <!--<div class="border"> -->
+    <div class="layout">
       <div class="tech">
-        <img class="img-tech" src="assets/react_icon.svg" alt="React Icon">
+        <img class="img-tech" src="assets/react_icon.svg" alt="tech Icon">
         <p class="technology-name">ReactJS</p>
         <p class="release">Release: 2013</p>
         <p class="description">React is a declarative, component-based JavaScript library for building user interfaces.</p>
       </div>
-    <!--</div>-->
+      <div class="tech">
+        <img class="img-tech" src="assets/angular_icon.svg" alt="tech Icon">
+        <p class="technology-name">AngularJS</p>
+        <p class="release">Release: 2010</p>
+        <p class="description">AngularJS (commonly referred to as AngularJS 1.X) is a JavaScript framework for building SPAs with an MVC architechture.</p>
+      </div>
+      <div class="tech">
+        <img class="img-tech" src="assets/vue_icon.svg" alt="tech Icon">
+        <p class="technology-name">VueJS</p>
+        <p class="release">Release: 2014</p>
+        <p class="description">VueJS (pronounced /vju:/, like view) is a open-source progressive JavaScript framework for building user interfaces.</p>
+      </div>
+      <div class="tech">
+        <img class="img-tech" src="assets/elm_icon.svg" alt="tech Icon">
+        <p class="technology-name">Elm</p>
+        <p class="release">Release: 2016</p>
+        <p class="description">Elm is a domain-specific functional programming languaje for declaratively creating web browser-based graphical user interfaces.</p>
+      </div>
+      <div class="tech">
+        <img class="img-tech" src="assets/polymer_icon.svg" alt="tech Icon">
+        <p class="technology-name">Polymer</p>
+        <p class="release">Release: 2015</p>
+        <p class="description">Polymer is a open-source JavaScript library for building web applications using Web Components.</p>
+      </div>
+    </div>
+    <h2 class="title">Back-End Tools</h2>
```

Comment:
Back end tools must have the same title importance of frontend tools. So replace this h2 for h1